tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -138,6 +138,51 @@ steps:
                   exit 1
                 fi
             else
+              feedId=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
+
+              if [[ $release == "release" && $feedId == "build" ]]; then
```

Comment:
Is the plan to add retention policy on "internal" feed? Or we are not adding it because we assume it can always upstream from "build"?

Reply:
We did not add it because we assumed it can always upstream from "build". I tested promoting the same package on the internal feed in parallel as build. Even though the newly published package does not appear on the internal feed, the promotion will get applied to to the package on the internal feed once it does become available. so promoting the same package in internal feed parallel is feasible.